sections/semantics-sections.include

```diff
@@ -1029,7 +1029,9 @@
     <dd><a>Flow content</a>.</dd>
     <dd><a>Palpable content</a>.</dd>
     <dt><a>Contexts in which this element can be used</a>:</dt>
-    <dd>Where <a>flow content</a> is expected.</dd>
+    <dd>
+      Where <a>flow content</a> is expected, but with no address, dt, or th element ancestors, or header, footer elements that are not ancestors of sectioning content which is a ancestors of the header.
```

Comment:
I think it might be clearer to phrase this as follows:

where flow content is expected, but not within address, dt or th elements. Inside header/footer elements, only within a new sectioning element inside those elements.